elements/rh-tile/rh-tile-group.css

```diff
@@ -0,0 +1,8 @@
+:host([aria-disabled="true"]) {
```

Comment:
we should prefer our own `disabled` attr, not expose `aria-` as public apis

Reply:
@bennypowers There's an accessibility issue with using our own disabled as we want the tile to be readable by assistive tech just not actionable.